shortfin/python/shortfin/debug/array.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+import numpy as np
```

Comment:
Let's instead put this under shortfin.array.np or something like that. And then we only include that in the API if numpy is installed.

Reply:
Moved to shortfin.array.nputils